### PR TITLE
Client side

### PR DESCRIPTION
**Pull request by DylanHul**
thunk added to project

### CODE REVIEW

File: Client/actions/action.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            from_account_id: 1,
+            to_account_id: 2
+          },
+          {
+            id:504,
+            amount: 56700,
+            description: 'weekly pay from bk',
+            from_account_id: 2,
+            to_account_id: 1
+          }
```
Comment (DylanHul) [inline Client/actions/action.js]: this receive action should take an argument (data)

Comment (DylanHul) [inline Client/actions/action.js]: should build the payload from the data

File: Client/actions/action.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        ]
+      }
+    }
+  }
+}
+
+export function fetchAccountTransaction () {
```
Comment (DylanHul) [inline Client/actions/action.js]: not export here

File: Client/actions/action.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        if (err) {
+          console.error(err.message)
+          return
+        }
+        dispatch(retrieveData(res.body.data.account.transactions))
+      })
+  }
+}
+
+  module.exports={retrieveData}
```
Comment (DylanHul) [inline Client/actions/action.js]:
``` js
module.exports={
  retrieveData,
  fetchAccountTransaction
}

```

File: Client/actions/action.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+export function fetchAccountTransaction () {
+  return (dispatch) => {
+    request
+      .get('/api/v1/accounts/:id/transactions')
+      .end((err, res) => {
+        if (err) {
+          console.error(err.message)
+          return
+        }
+        dispatch(retrieveData(res.body.data.account.transactions))
```
Comment (DylanHul) [inline Client/actions/action.js]: this needs to be `receiveAccountTransaction`

File: Client/index.js [hunk 5]
```diff
@@ -3,17 +3,23 @@ import ReactDOM from 'react-dom'
...
 
 
 
 domready(() => {
   ReactDOM.render(
     <div>
     <h1>Here</h1>
-    <Provider>
+    <Provider store={store}>
       <App/>
```
Comment (DylanHul) [inline Client/index.js]: unnecessary divs /  bad indenting

**Comment by JamesAnBo** [comment]
I like your style! 🍠  I'll go ahead and merge.

**Pull request merged**